.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
We need to mention that the PR needs to follow these commit guidelines: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines. Specifically the title needs to `<type>(<scope>): <subject>`